Makefile

```diff
@@ -42,6 +42,12 @@ $(PACKAGE_SCRIPTS) \
 --config-files "/etc/$(BINARY)/$(CONFIG_FILE)"
 endef
 
+VERSION_LDFLAGS:= \
+  -X $(IMPORT_PATH)/vendor/github.com/prometheus/common/version.Branch=$(BRANCH) \
+  -X $(IMPORT_PATH)/vendor/github.com/prometheus/common/version.BuildDate=$(DATE) \
+  -X $(IMPORT_PATH)/vendor/github.com/prometheus/common/version.Revision=$(COMMIT) \
```

Comment:
Does this seem OK though? You mentioned you'd rather not do this. I'm on the fence. I like the idea of containerizing the pieces in one package we can import from anywhere (ie, if want to expose these variables to the influxunifi or promunifi packages, it works). I don't like that `vendor/` is in this path though, that bugs me. I haven't looked into go modules yet, but I feel like this wont work if someone uses them. Maybe I shouldn't care about that and cross the bridge when we get there. What's your feeling on this?

Reply:
...and I just saw your other comment about how this works with go modules. I'm fine either way you want to swing this, but I'm kinda liking it _this_ way.

Reply:
Yea, this is fine, it's easy to adjust the `-X` flag if the build process ever changes.